src/components/configure/configuration/ConfigurationDialog.tsx

```diff
@@ -209,6 +212,28 @@ export default class ConfigurationDialog extends React.Component<
     }
   }
 
+  private onClickDownloadCheatSheet() {
+    const keymaps: { [pos: string]: IKeymap }[] = this.props.keymaps!;
+    const keys: { [pos: string]: Key }[] = [];
+    for (let i = 0; i < this.props.layerCount!; i++) {
+      const keyMap: { [pos: string]: Key } = {};
+      const km = keymaps[i];
+      Object.keys(km).forEach((pos) => {
+        const key: Key = genKey(km[pos], 'en-gb');
+        keyMap[pos] = key;
+      });
+      keys.push(keyMap);
+    }
+
+    const pdf = new KeymapPdfGenerator(
+      this.props.keyboardDefinition!.layouts.keymap,
+      keys,
+      this.props.layerCount!,
+      'en-gb'
+    );
+    pdf.genPdf(this.props.productName!);
+  }
```

Comment:
@adamrocker I guess that the purpose of this code is for testing during your development, therefore this method should be removed.